src/components/vm/nics/nicBody.tsx

```diff
@@ -271,3 +304,363 @@ export const NetworkTypeAndSourceRow = ({
         </>
     );
 };
+
+export function portForwardText(pf: VMInterfacePortForward): string {
+    let text = "";
+
+    if (pf.address)
+        text += pf.address + ":";
+    if (pf.dev)
+        text += pf.dev + ":";
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-3e464b33-20251031-114223-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)